released/packages/coq-hierarchy-builder/coq-hierarchy-builder.0.9.0/opam

```diff
@@ -0,0 +1,24 @@
+opam-version: "2.0"
+name: "coq-hierarchy-builder"
+version: "dev"
+maintainer: "Enrico Tassi <enrico.tassi@inria.fr>"
+authors: [ "Cyril Cohen" "Kazuhiko Sakaguchi" "Enrico Tassi" ]
+license: "MIT"
+homepage: "https://github.com/math-comp/hierarchy-builder"
+bug-reports: "https://github.com/math-comp/hierarchy-builder/issues"
+dev-repo: "git+https://github.com/math-comp/hierarchy-builder"
+
+build: [ make ]
```

Comment:
```suggestion
build: [ make "-j%{jobs}%" ]
```
Is there a reason for not building in parallel?

Reply:
It takes only a few seconds, but I believe it works